### ISSUE TITLE
Apply command doesn't respect branches on transform

### ISSUE THREAD

**Issue by ParamagicDev** [bridgetownrb/bridgetown#65]
When supplying a branch such as "automation" or "development" to the `bridgetown apply` command, it appends "master"

**Bridgetown Version**:

0.15.0.beta-2

**Current Behavior**:

```bash
bridgetown apply https://github.com/ParamagicDev/bridgetown-plugin-tailwindcss/tree/automation/bridgetown.automation.rb
```

Turns 

"https://github.com/ParamagicDev/bridgetown-plugin-tailwindcss/tree/automation/bridgetown.automation.rb" 

into 

"https://raw.githubusercontent.com/ParamagicDev/bridgetown-plugin-tailwindcss/automation/master/bridgetown.automation.rb"

**Expected Behavior**

"https://github.com/ParamagicDev/bridgetown-plugin-tailwindcss/tree/automation/bridgetown.automation.rb" 

Should turn to

"https://raw.githubusercontent.com/ParamagicDev/bridgetown-plugin-tailwindcss/automation/bridgetown.automation.rb"

**Comment by jaredcwhite** [comment]
Thanks @ParamagicDev — the fancy GitHub URL parsing is indeed not accounting for paths in non-master branches. Good catch.

**Comment by jaredcwhite** [comment]
Actually, if you're feeling inspired, maybe you could add a PR with some failing examples in the test (`bridgetown-core/test/test-apply-command.rb`), and I'd love to pair on fixes

**Comment by ParamagicDev** [comment]
I added a test for it...and i couldnt help but solve the issue as well...I submitted a PR

#66

**Comment by jaredcwhite** [comment]
Done, now available in 0.15.0.beta3 public release.
